src/asn1/helpers.h

```diff
@@ -0,0 +1,85 @@
+#ifndef ASN1_HELPERS_H_
+#define ASN1_HELPERS_H_
+
```

Comment:
You need to add all source files to "aktualizr_source_file_checks" function in cmake. Now, `make format` and `make check` don't checks this file. Also implementation should be in cc file, only inline implementation is allowed in header files.